src/rpc/blockchain.cpp

```diff
@@ -2319,6 +2319,41 @@ UniValue scantxoutset(const JSONRPCRequest& request)
     return result;
 }
 
+static UniValue getlicensedminers(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0)
+        throw std::runtime_error(
+            RPCHelpMan{"getlicensedminers",
+                "\nReturns the list of licensed miners.\n",
+                {},
+				RPCResult{
+				            "\"licenses\": [\n"
+				            "    {\n"
+				            "      \"height\": n,                  (numeric) Height of the last modifiaction of given license\n"
+				            "      \"hashrate\": n,                (numeric) Hashrate assigned to miner where 1 assigned unit is 10 PH\n"
```

Comment:
I suggest to name the variable as a self-explanatory `hashrate_ph` and show real hashrate in PH

Reply:
`Hashrate assigned to a miner`

Reply:
Done